mozilla-release/docshell/base/nsDocShell.cpp

```diff
@@ -9180,6 +9180,11 @@ nsresult nsDocShell::InternalLoad(nsDocShellLoadState* aLoadState,
       if (uri->SchemeIs("moz-extension") && uri->GetSpecOrDefault().Equals(connectURLString)) {
         break;
       }
+      nsCString datURLString;
```

Comment:
Should these lines be wrapped with a comment to indicate this is custom for Cliqz?

Reply:
This follows the Cliqz special code for connect that has a preceding comment